x-pack/plugins/endpoint/public/applications/endpoint/view/alerts/index.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { memo, useState, useMemo } from 'react';
+import React from 'react';
+import { EuiDataGrid } from '@elastic/eui';
+import { useSelector } from 'react-redux';
+import * as selectors from '../../store/selectors';
+import { usePageId } from '../use_page_id';
+
+export const AlertIndex = memo(() => {
+  usePageId('alertsPage');
+
+  const columns: Array<{ id: string }> = [
+    { id: 'alert_type' },
+    { id: 'event_type' },
+    { id: 'os' },
+    { id: 'ip_address' },
+    { id: 'host_name' },
+    { id: 'timestamp' },
+    { id: 'archived' },
+    { id: 'malware_score' },
+  ];
+
+  const [visibleColumns, setVisibleColumns] = useState(() => columns.map(({ id }) => id));
+
+  const json = useSelector(selectors.alertListData);
+
+  const renderCellValue = useMemo(() => {
+    return ({ rowIndex, columnId }: { rowIndex: number; columnId: string }) => {
+      if (json.length === 0) {
+        return null;
+      }
+
+      if (columnId === 'alert_type') {
+        return json[rowIndex].value.source.endgame.metadata.key;
+      } else if (columnId === 'event_type') {
+        return json[rowIndex].value.source.endgame.data.file_operation;
```

Comment:
Can you confirm that all alerts are guaranteed to have this field?

Reply:
We will do this properly in future PRs. We just want to get some basic code structure and patterns merged now.